src/extension.ts

```diff
@@ -306,16 +342,13 @@ class ProbeRsDebugAdapterTracker implements DebugAdapterTracker {
 	}
 
 	onError(error: Error) {
-		if (probeRsLogLevel === 'Debug')
-		{
+		if (probeRsLogLevel === 'Debug') {
```

Comment:
What happens if the log leve is 'trace'? I would assume that `trace` includes everything from debug and more, but here the message would not be shown in that case.

Reply:
Good catch.